src/portal/secret.rs

```diff
@@ -10,9 +10,29 @@ use rand::{distributions::Alphanumeric, thread_rng, Rng};
 #[cfg(feature = "tokio")]
 use tokio::{io::AsyncReadExt, net::UnixStream};
 use zbus::zvariant::{Fd, ObjectPath, OwnedValue, SerializeDict, Type};
+use zeroize::{Zeroize, ZeroizeOnDrop};
 
 use super::Error;
 
+#[derive(Debug, Zeroize, ZeroizeOnDrop)]
+pub struct Secret {
```

Comment:
As Secret is now part of the public API, maybe it needs to be re-exported in src/portal/mod.rs? or event src/lib.rs?

Reply:
Also could use a small comment about what this type represents

Reply:
> maybe it needs to be re-exported in src/portal/mod.rs

It is re-exported in `src/portal/mod.rs` already. I added docs to it.